From 4899999f32242162a10a4893f920e796234983f0 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 7 Nov 2017 16:33:55 +0100
Subject: [PATCH 0236/1852] v4l: xilinx: sdirxss: Remove useless semicolon

commit 59c6578dfdbaeaf43a20124c96b5c4b1319ff0bb from
https://github.com/Xilinx/linux-xlnx.git

Remove error reported by coccinelle:
drivers/media/platform/xilinx/xilinx-sdirxss.c:791:3-4: Unneeded
semicolon

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-sdirxss.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-sdirxss.c b/drivers/media/platform/xilinx/xilinx-sdirxss.c
index 9440bd47f748..5e0d09ca2f69 100644
--- a/drivers/media/platform/xilinx/xilinx-sdirxss.c
+++ b/drivers/media/platform/xilinx/xilinx-sdirxss.c
@@ -788,7 +788,7 @@ static int xsdirx_get_stream_properties(struct xsdirxss_state *state)
 		default:
 			dev_dbg(core->dev, "Unknown 12G Mode SMPTE standard\n");
 			return -EINVAL;
-		};
+		}
 		break;
 	default:
 		dev_err(core->dev, "Invalid Mode\n");
-- 
2.31.1

